operators/cv2/opencv.cc

```diff
@@ -1,10 +1,10 @@
 #include "ocos.h"
-#include "gaussian_blur.hpp"
+#include "no_codecs/gaussian_blur.hpp"
 #ifdef ENABLE_OPENCV_CODECS
-#include "imread.hpp"
-#include "imdecode.hpp"
-#include "super_resolution_preprocess.hpp"
-#include "super_resolution_postprocess.hpp"
+#include "codecs/imread.hpp"
+#include "codecs/imdecode.hpp"
+#include "codecs/super_resolution_preprocess.hpp"
```

Comment:
Do we still need them, or we can get rid of them now? There is no production usage right now and was only developed for Mobile.

Reply:
I'll remove the super_resolution_* ones as they aren't needed. Do you want imread/imdecode left or deleted as well?

Reply:
probably we can undo the previous PR to clean up test and tutorial files as well in another PR?

there are some nice python tests based on imread/imdecodec that help evaluating the openCV library. Let's keep it for a while.